src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -52,10 +52,10 @@
 import java.nio.*;
 import java.time.*;
 import java.util.*;
-import java.util.stream.*;
 import java.util.concurrent.*;
 import java.util.concurrent.atomic.*;
 import java.util.function.*;
+import java.util.stream.*;
 
```

Comment:
Are these changes just IntelliJ having fun?

Reply:
Yeah, I ran optimize imports because I thought I had removed something from here and wanted to get rid of it, but guess it did some ordering tweaks instead.